catalyst/utils/config.py

```diff
@@ -47,10 +47,19 @@ def save_config(config, logdir: str) -> None:
         json.dump(config, fout, indent=2)
 
 
+def set_arg_to_config(config, key, value) -> None:
```

Comment:
I think, you need to check, that value is not `None`, cause you can corrupt config by parser defaults.

Reply:
i think if we pass a value to config it should overwrite it anyway

Reply:
okay, let me explain the full story:
1. you define argparse with some arguments like `--n-workers`, `--batch-size`, etc
2. by default they are None
3. but you have defined them in `config/args`
4. because it's easier to define them in config, rather than pass them every train run

5. you PR feature to transfer all args kwargs to config...you rewrite the config with default args values (None) from parser -> conflict, checks have failed

Reply:
hope fix should help :)

Reply:
Thanks, got it